tests/support/pretest.js

```diff
@@ -110,25 +113,29 @@ function getHarness(tree) {
 }
 
 function modFiles() {
-	return glob(path.join(testPath, '*.html')).then(function(paths) {
+	return glob(path.join(testPath, '**/*.html')).then(function(paths) {
 		return Promise.all(_.map(paths, function(path) {
 			return fs.readFileAsync(path, 'utf8')
-				.then(modFile)
+				.then(function(source) {
+					return modFile(source, path);
+				})
 				.then(function(source) {
 					return fs.writeFileAsync(path, source, 'utf-8');
 				});
 		}));
 	});
 }
 
-function modFile(source) {
+function modFile(source, filePath) {
+	var fileDir = path.dirname(filePath);
+	var pepPath = path.relative(fileDir, path.join(basePath, 'dist', 'pep.js'));
```

Comment:
The iframe HTML files in the `tests/w3c/resources` have a different relative path to `dist/pep.js` than the files `tests/w3c`.